docs/modules/Record.ts.md

```diff
@@ -135,20 +175,49 @@ export declare const filter: {
 }
 ```
 
+**Example**
+
+```ts
+import { filter } from 'fp-ts/Record'
+
+assert.deepStrictEqual(filter((s: string) => s.length < 4)({ a: 'foo', b: 'bar', c: 'verylong' }), {
+  a: 'foo',
+  b: 'bar',
+})
+```
+
 Added in v2.0.0
 
 ## filterMap
 
+Maps a `Record` with an iterating function that returns an option
+and it keeps only the `option.some` values.
+
 **Signature**
 
 ```ts
 export declare const filterMap: <A, B>(f: (a: A) => Option<B>) => (fa: Record<string, A>) => Record<string, B>
 ```
 
+**Example**
+
+```ts
+import { filterMap } from 'fp-ts/Record'
+import { option } from 'fp-ts'
+
+const f = (s: string) => (s.length < 4 ? option.some(`${s} is short`) : option.none)
+assert.deepStrictEqual(filterMap(f)({ a: 'foo', b: 'bar', c: 'verylong' }), {
+  a: 'foo is short',
+  b: 'bar is short',
+})
+```
+
 Added in v2.0.0
 
 ## partition
 
+Partition a `Record` into 2 parts according to a predicate.
```

Comment:
I most often see numbers under ten or so referred to in words i.e. "two" rather than "2". I'd apply the same remark to every other instance.